website/content/docs/upgrading/upgrade-to-1.15.x.mdx

```diff
@@ -82,3 +82,5 @@ option.
 @include 'known-issues/perf-standbys-revert-to-standby.mdx'
 
 @include 'known-issues/1_13-reload-census-panic-standby.mdx'
+
+@include `known-issues/autopilot-upgrade-upgrade-version.mdx'
```

Comment:
```suggestion
@include 'known-issues/autopilot-upgrade-upgrade-version.mdx'
```

Reply:
if you check the vercel build, you can see that the partial wasn't getting loaded properly :) 
<img width="640" alt="image" src="https://github.com/hashicorp/vault/assets/29907172/7db024ca-516d-4f6e-a476-313d150408fb">

Reply:
Ahh, thanks, fixed.